DataProcessing/Crop.py

```diff
@@ -0,0 +1,49 @@
+import os
+import torch
+import numpy as np
+
+
+class Crop:
+    def __init__(self, frame_id: int,
+                 bbox: np.array,
+                 crop_img: torch.tensor,
+                 face_img: torch.tensor,
+                 track_id: int,
+                 cam_id: int,
+                 crop_id: int,
+                 video_name: str,
+                 is_face : bool
+                 ):
+        self.frame_id = int(frame_id)
+        self.bbox = bbox
+        self.crop_img = crop_img
+        self.face_img = face_img
+        self.track_id = int(track_id)
+        self.cam_id = int(cam_id)
+        self.crop_id = int(crop_id)
+        self.video_name = video_name # note the name must be already 'clean' prior to crop Init
+        self.label = None
+        self.unique_crop_name = None
+        self.update_hash()
+        self.is_face = is_face
+        self.is_vague = False
+
+    def set_label(self, label):
+        self.label = label
+        self.update_hash() # update unique_hash
```

Comment:
remove this line since the label isn't a part of the file name anymore so we don't need to update it once it changes